src/pages/user-projects/user-projects.less

```diff
@@ -0,0 +1,7 @@
+#userProjects {
+  display: block;
+  position: relative;
+  width: 100%;
+  height: 100%;
+  padding: 20px 0;
+}
```

Comment:
The CSS here is the same as `#discover` in `discover.less`. 

Can we just share a common class between them?